package.json

```diff
@@ -5,32 +5,31 @@
   "main": "AbstractSQLCompiler.js",
   "types": "AbstractSQLCompiler.d.ts",
   "scripts": {
-    "pretest": "npm run prepublishOnly",
+    "pretest": "npm run prepare",
     "test": "mocha",
     "prepublish": "require-npm4-to-publish",
-    "prepublishOnly": "tsc && ometajs2js --input AbstractSQLOptimiser.ometajs --output AbstractSQLOptimiser.js && ometajs2js --input AbstractSQLRules2SQL.ometajs --output AbstractSQLRules2SQL.js"
+    "prepare": "tsc && ometajs2js --input AbstractSQLOptimiser.ometajs --output AbstractSQLOptimiser.js && ometajs2js --input AbstractSQLRules2SQL.ometajs --output AbstractSQLRules2SQL.js"
   },
   "repository": "https://github.com/resin-io-modules/abstract-sql-compiler.git",
   "author": "",
   "dependencies": {
     "@resin/sbvr-types": "^1.4.2",
-    "@types/bluebird": "3.5.20",
-    "@types/lodash": "4.14.109",
-    "@types/node": "8.10.19",
+    "@types/bluebird": "^3.5.24",
+    "@types/lodash": "^4.14.116",
```

Comment:
I remember that at some point bluebird and lodah typings were getting patch releases that broke our builds.
I guess that since we are adding back the `^` they don't do that any more.

Reply:
Yeah, I haven't noticed it recently at all, what I have definitely noticed is using fixed bluebird versions causing `type Bluebird is incompatible with type Bluebird` (or a version of that) due to having multiple versions of the bluebird types in a project that haven't been deduped